datadog/init.sls

```diff
@@ -11,13 +11,17 @@ datadog_pkg_dependencies:
 
 datadog_install:
   cmd.run:
-    - name: 'DD_AGENT_MAJOR_VERSION={{ datadog.major_version }} DD_API_KEY=api_key DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"'
+    - name: curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh
```

Comment:
This is just downloading the script file itself. This would probably be easiest as a `cmd.script` execution. You can just set the source parameter as the s3 location and remove the `name` parameter.